index.html

```diff
@@ -3,20 +3,11 @@
 <html>
 <head>
 	<title>ES6 Classes</title>
-	<script src="./dist/canvanoid.bundle.js"></script> 
+	
 </head>
 <body>
 	<canvas id="canvas"></canvas>
-	<script> //"babel src -d dist",
-		window.requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame || 
-									window.msRequestAnimationFrame || window.mozRequestAnimationFrame;
-					
-		var canvas = document.getElementById('canvas')
-		canvas.width = window.innerWidth - 100; 
-		canvas.height = window.innerHeight - 100; 
 
-		var game = new Canvanoid(canvas); 	
-		game.start();
-	</script>
+	<script src="./dist/canvanoid.bundle.js"></script> 
```

Comment:
mmm, I use it the other way around, when you deploy your site the bundle js file will be kept in the browser cache, rather add some cache breaking mechanism to your bundle.js since you don't have control over the browser or a proxy server caching you js file.